src/workerd/io/script-version.capnp

```diff
@@ -8,10 +8,15 @@ using Cxx = import "/capnp/c++.capnp";
 $Cxx.namespace("workerd");
 
 struct ScriptVersion {
-  id @0 :Text;
-  # An ID that should be used to uniquely identify this version.
-  tag @1 :Text;
+  id :group {
+    upper @0 :UInt64;
+    # Most significant bits of the UUID.
+    lower @1 :UInt64;
+    # Least significant bits of the UUID.
+  }
```

Comment:
Is this struct unused in production? if it is used, couldn't we end up with errors when the old struct layout is parsed while this new one is expected?

Reply:
This has never been used in production, it's safe to change.